package.json

```diff
@@ -7,9 +7,8 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "npm run lint && jest",
-    "eslint:check": "eslint --print-config ./src/index.ts | eslint-config-prettier-check",
```

Comment:
The CLI tool for checking compatible rules show now be run like this:

```
eslint-config-prettier src/index.ts
```

Reply:
Fixed this. I looked the docs yesterday but was thinking that there is something wrong with this.